packages/common/http/src/client.ts

```diff
@@ -173,7 +173,7 @@ export class HttpClient {
    * @param url     The endpoint URL.
    * @param options The HTTP options to send with the request.
    *
-   * @return An `Observable` of the response, with the response body as an array of `HTTPEvents` for
+   * @return An `Observable` of the response, with the response body as an array of `HttpEvents` for
    *     the
    * request.
```

Comment:
While you are at it, feel free to get rid of the unnecessary newline here:

```suggestion
   *     the request.
```